docs/man_bodhi.rst

```diff
@@ -243,6 +243,58 @@ The ``updates`` command allows users to interact with bodhi updates.
 
     Request that the given update be changed to the given state. ``update`` should be given by
     update id, and ``state`` should be one of testing, stable, unpush, obsolete, or revoke.
+    
+    
+    
+Examples
+========
+
+Some common examples of bodhi CLI operations:
+
+    ``$ bodhi --new --type bugfix bodhi-0.4.2-1.fc7``
```

Comment:
I think these examples are from bodhi-0.9 instead of bodhi-2.2. For example, in bodhi-2.2 the new update command would look like:

```
$ bodhi updates new --type bugfix bodhi-2.2.4-1.fc26
```